Cargo.toml

```diff
@@ -45,14 +45,19 @@ serde_json = { version = "1.0" }
 toml = "0.5.8"
 serde_derive = "1.0"
 serde = "1.0.130"
-rusqlite = "0.29.0"
+rusqlite = { version = "0.29.0", features = ["bundled"] }
 simplelog = "0.7.1"
 dirs = "3.0.1"
 log = "0.4.14"
 staking_credentials = { git = "https://github.com/civkit/staking-credentials.git", branch = "main" }
 reqwest = "0.11.20"
 base64 = "0.21.4"
 jsonrpc = "0.14.0"
+rs_merkle = "1.4.1"
+bitcoincore-rpc = "0.17.0"
```

Comment:
I think we can remove the usage of this crate. Latest branch introduce a functional RPC client, gives us more flexibility on parameters formatting.